### ISSUE TITLE
[Narrator] Narrator focus is not coming inside the Bot Framework Emulator V4 Application.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#727]
**Actual Result:**
Narrator focus is not coming inside the Bot Framework Emulator V4 Application.

**Expected Result:**
Narrator focus should come inside the Bot Framework Emulator V4 Application.

**NOTE:**
This behavior is intermittent, Narrator stops working in the application when we open any bot and then starts working only when we restart the system.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Verify that live chat tab is opened.
4. Start Narrator using Ctrl + Windows logo key + Enter.
5. Now navigate in application using Tab or Up/Down arrow keys.
6. Check if Narrator focus is coming in the application or not.

**User Impact:**
Narrator users won't be able to use Bot Framework Emulator V4 Application if this behavior is there.

Test Environment:
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge
Tools: Narrator

Attachments:
[NARRATOR_BotFrameworkEmulator_Blocker_NarratorNotWorking.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2257541/NARRATOR_BotFrameworkEmulator_Blocker_NarratorNotWorking.zip)

**Comment by tonyanziano** [comment]
Windows Narrator is currently known not to work with Chromium / Chrome.

**Comment by mike-lischke** [comment]
I'd like to add that accessibility is not only about screen readers (and text-to-speech support). Correct accessibility support includes the ability to identify every UI element individually and allow for certain actions (like reading/setting text in an editor area, scroll a scroll area, press a button etc.). And in this scenario vscode fails badly. Use any of the accessibility inspection tools (e.g. `Accessibility Inspector`, which comes with XCode) to view details of vscode. All you will get is the top level window, even if that socalled "screen reader optimization" is active. Compare that to any other desktop application to see what vscode is missing in this area.
